tests/server/api/api_test.go

```diff
@@ -117,12 +120,14 @@ func (s *serverTestSuite) TestReconnect(c *C) {
 var _ = Suite(&testMiddlewareSuite{})
 
 type testMiddlewareSuite struct {
-	cleanup func()
-	cluster *tests.TestCluster
+	cleanup        func()
+	cluster        *tests.TestCluster
+	tempStdoutFile *os.File
 }
 
 func (s *testMiddlewareSuite) SetUpSuite(c *C) {
 	c.Assert(failpoint.Enable("github.com/tikv/pd/server/api/enableFailpointAPI", "return(true)"), IsNil)
+	s.tempStdoutFile, _ = os.CreateTemp("/tmp", "pd_tests")
```

Comment:
Perhaps, moving it into the specific test case is better

Reply:
has moved it